tools/ci_set_matrix.py

```diff
@@ -224,18 +212,15 @@ def get_settings(board):
             boards_to_build = all_board_ids
             break
 
-    # Split boards by job
-    job_to_boards = {"aarch": [], "arm": [], "atmel": [], "esp": [], "riscv": []}
-
     # Append previously failed boards
-    for job in job_to_boards:
-        if job in last_failed_jobs:
-            for board in last_failed_jobs[job]:
-                boards_to_build.add(board)
+    last_failed_boards = last_failed_jobs.get("boards")
+    if last_failed_boards:
+        boards_to_build.update(last_failed_boards)
 
-    build_boards = bool(boards_to_build)
-    print("Building boards:", build_boards)
-    set_output("build-boards", build_boards)
+    print("Building boards:", bool(boards_to_build))
```

Comment:
Why bool?

Reply:
This just logs if any boards are being built.